Apsitvarkom.IntegrationTests/DataAccess/DatabaseTests.cs

```diff
@@ -73,4 +73,17 @@ public async Task GetByPropertyTest()
             Assert.That(response.Notes, Is.EqualTo(dbRow.Notes));
         });
     }
+
+    [Test]
+    public async Task InsertAsync_SamePrimaryKeys_Throws()
```

Comment:
We could also test for successful addition.

Reply:
Happy path is handled in Unit tests, and this scenario could only be tested with an in-memory db, so that's why it's there. We could have more tests in the future (where validation fails, etc.), but I would prefer if we did it with another task, maybe someone else from the team wants to chip in on this

Reply:
Fair.